src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -25,36 +29,35 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Controllers
     /// Controller responsible for handling all administrative requests, for
     /// example enqueueing function invocations, etc.
     /// </summary>
-    [JwtAuthentication]
-    [AuthorizationLevel(AuthorizationLevel.Admin)]
```

Comment:
Do we not support a class level annotation anymore? This class is named AdminController implying that all operations by default require admin level. You're inverting that?

Reply:
It is supported, but since we have different policy requirements for some of the actions, moving them to the method was more flexible.